webrtc-stats.html

```diff
@@ -768,6 +768,7 @@
              double             burstDiscardRate;
              double             gapLossRate;
              double             gapDiscardRate;
+             DOMHighResTimeStamp  lastPacketReceivedTimestamp;
```

Comment:
In the local reports, the timestamp indicates when the report was generated.
In the remote reports, the timestamp indicates when the report was received.

I always thought that the timestamp in the getStats() for local objects was timestamp when the snapshot of the getStats was created, which may have been when the getStats was called.